datafusion/execution/src/runtime_env.rs

```diff
@@ -89,6 +89,27 @@ impl RuntimeEnv {
     /// scheme, if any.
     ///
     /// See [`ObjectStoreRegistry`] for more details
+    ///
+    /// # Examples
+    ///
+    /// Here's how to set up a local file system object store:
+    /// ```
+    /// let url = Url::try_from("file://").unwrap();
+    /// let object_store = object_store::local::LocalFileSystem::new();
+    /// ctx.runtime_env()
+    ///    .register_object_store(&url, Arc::new(object_store));
+    /// ```
+    ///
+    /// Or how to set up a http object store:
+    /// ```
+    /// let base_url = Url::parse("https://github.com").unwrap();
```

Comment:
These examples are run automatically via `cargo test --doc` -- so they need to be completely standlone

I took the liberty of pushing a commit to this PR that did so

Reply:
Thanks so much for making these pass, I'll remember to run the doc tests next time, only looked at the doc site before submitting the PR.